boxsdk/client/client.py

```diff
@@ -742,6 +742,76 @@ def create_group(
             response_object=response,
         )
 
+    def terms_of_service(self, tos_id):
+        """
+        Initialize a :class:`TermsOfService` object, whose box id is tos_id.
+
+        :param tos_id:
+            The box id of the :class:`TermsOfService` object.
+        :type tos_id:
+            `unicode`
+        :return:
+            A :class:`TermsOfService` object with the given terms of service id.
+        :rtype:
+            :class:`TermsOfService`
+        """
+        return self.translator.get('terms_of_service')(session=self._session, object_id=tos_id)
+
+    def terms_of_service_user_status(self, tos_user_status_id):
+        """
+        Initialize a :class:`TermsOfServiceUserStatus` object, whose box id is tos_user_status_id.
+
+        :param tos_user_status_id:
+            The box id of the :class:`TermsOfServiceUserStatus` object.
+        :type tos_id:
+            `unicode`
+        :return:
+            A :class:`TermsOfServiceUserStatus` object with the given terms of service user status id.
+        :rtype:
+            :class:`TermsOfServiceUserStatus`
+        """
+        return self.translator.get('terms_of_service_user_status')(session=self._session, object_id=tos_user_status_id)
+
+    def get_terms_of_services(self, tos_type=None, limit=None, marker=None, fields=None):
+        """
+        Get the entries in the legal hold policy assignment using limit-offset paging.
+
+        :param tos_type:
+            Can be set to `managed` or `external` for the type of terms of service.
+        :type tos_type:
+            `unicode`
+        :param: limit
+            The maximum number of items to return. If limit is set to None, then the default
+            limit (returned by Box in the response) is used. See https://developer.box.com/reference#get-recent-items
+            for default.
+        :type: limit
+            `int` or None
+        :param marker:
```

Comment:
This method can only return two total items today; I don't think the user could ever need to pass in a `marker`